htdocs/.htaccess

```diff
@@ -6,8 +6,8 @@
 RewriteRule ^([a-zA-Z_-]+)/ajax/([a-zA-Z0-9_.-]+)$ AjaxHelper.php?Module=$1&script=$2 [QSA,L]
 
 # Instruments and reliability still use main.php, for now
-RewriteRule ^instruments/([a-zA-Z0-9_-]+)/$ main.php?test_name=$1 [QSA,L]
-RewriteRule ^instruments/([a-zA-Z0-9_-]+)/([a-zA-Z0-9_.-]+)/$ main.php?test_name=$1&subtest=$2 [QSA,L]
+RewriteRule ^instruments/([a-zA-Z0-9_-]+)/$ main.php?test_name=$1 [QSA,END]
+RewriteRule ^instruments/([a-zA-Z0-9_-]+)/([a-zA-Z0-9_.-]+)/$ main.php?test_name=$1&subtest=$2 [QSA,END]
 
 RewriteRule ^([a-zA-Z0-9_-]+)reliability([a-zA-Z0-9_-]*) main.php?test_name=$1reliability$2 [QSA,L]
```

Comment:
would it make sense to change `L` here as well?

Reply:
@davidblader I think END supersede L but I will check again

Reply:
@davidblader END is almighty
from doc:
> An alternative flag, [END], can be used to terminate not only the current round of rewrite processing but prevent any subsequent rewrite processing from occurring in per-directory (htaccess) context.

Reply:
right, i am just wondering if the same flags need to be set for reliability, since i don't have any way to test that
i think we should get an IBIS developer ( @sruthymathew123 , @llevitis , @suzanne-lee ) to make sure everything in reliability is working as well

Reply:
Should probably be done for AjaxHelper too